src/GitHub.App/Services/PullRequestService.cs

```diff
@@ -20,5 +21,28 @@ public IObservable<IPullRequestModel> CreatePullRequest(IRepositoryHost host, IS
 
             return host.ModelService.CreatePullRequest(repository, title, body, source, target);
         }
+
+        public string GetPullRequestTemplate(ISimpleRepositoryModel repository)
+        {
+            Extensions.Guard.ArgumentNotNull(repository, nameof(repository));
+
+            var paths = new[]
+            {
+                Path.Combine(repository.LocalPath, "PULL_REQUEST_TEMPLATE"),
+                Path.Combine(repository.LocalPath, "PULL_REQUEST_TEMPLATE.md"),
+                Path.Combine(repository.LocalPath, ".github", "PULL_REQUEST_TEMPLATE"),
+                Path.Combine(repository.LocalPath, ".github", "PULL_REQUEST_TEMPLATE.md"),
+            };
+
+            foreach (var path in paths)
+            {
+                if (File.Exists(path))
+                {
+                    try { return File.ReadAllText(path); } catch { }
+                }
+            }
+
+            return null;
```

Comment:
I return `null` because they are two different outcomes: an empty string may be caused by an empty `PULL_REQUEST_TEMPLATE`, but null means "the pull request template was not found". This may be relevant if we decide we also want auto-population of the message from the last commit as in GHfW.